### ISSUE TITLE
WCAG 3.1.1: <html> element must have a lang attribute (#_yuiResizeMonitor;html)

### ISSUE THREAD

**Issue by hash-logic** [hash-logic/issue#4]
**Issue**: `<html> element must have a lang attribute` ([`html-has-lang`](https://dequeuniversity.com/rules/axe/3.2/html-has-lang?application=msftAI))

**Target application**: [Home · Custom Portal](https://mymatterssit.microsoftcrmportals.com/)

**Element path**: #_yuiResizeMonitor;html

**Snippet**:

    <html>

**How to fix**:

    Fix any of the following:
      The <html> element does not have a lang attribute

**Environment**:
Chrome version 73.0.3683.86

====

This accessibility issue was found using Accessibility Insights for Web 2.1.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
